src/player.cpp

```diff
@@ -13448,11 +13432,11 @@ Creature::Attitude player::attitude_to( const Creature &other ) const
     return A_NEUTRAL;
 }
 
-bool player::sees( const tripoint &t, int &bresen1, int &bresen2 ) const
+bool player::sees( const tripoint &t, int &/*bresen1*/, int &/*bresen2*/ ) const
 {
     static const std::string str_bio_night("bio_night");
     const int wanted_range = rl_dist( pos3(), t );
-    bool can_see = Creature::sees( t, bresen1, bresen2 );
+    bool can_see = g->m.pl_sees( t, wanted_range );
```

Comment:
`player::sees` is inherited by the npc class (and not overridden), wouldn't that make NPCs use `pl_sees` and thereby the player-visibility cache? And `pl_sees` has an explicit reference to `g->u`.